src/SprykerEco/Yves/FactFinderWebComponents/Plugin/Provider/FactFinderWebComponentsControllerProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Plugin\Provider;
+
+use Silex\Application;
+use Spryker\Yves\Application\Plugin\Provider\YvesControllerProvider;
+
+class FactFinderWebComponentsControllerProvider extends YvesControllerProvider
+{
+    /**
+     * @param \Silex\Application $app
+     *
+     * @return void
+     */
+    protected function defineControllers(Application $app)
+    {
+        $this->createController(
+            '/fact-finder-web-components',
+            'fact-finder-web-components',
```

Comment:
it would be great if we can have it as constants

Reply:
The string replaced with a constant.